bundleserver/src/main/java/net/discdd/server/service/BundleServerEchangeServiceImpl.java

```diff
@@ -0,0 +1,79 @@
+package net.discdd.server.service;
+
+import net.devh.boot.grpc.server.service.GrpcService;
+import net.discdd.bundlerouting.service.BundleExchangeServiceImpl;
+import net.discdd.grpc.BundleSender;
+import net.discdd.grpc.BundleSenderType;
+import net.discdd.server.bundletransmission.BundleTransmission;
+import org.springframework.beans.factory.annotation.Value;
+
+import javax.annotation.PostConstruct;
+import java.nio.file.Path;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import static java.util.logging.Level.SEVERE;
+
+@GrpcService
+public class BundleServerEchangeServiceImpl extends BundleExchangeServiceImpl {
```

Comment:
BundleServerEchangeServiceImpl - class name is spelled wrong, missing x in exchange